Kickstarter-iOS/Views/Cells/SettingsTextInputCell.swift

```diff
@@ -0,0 +1,121 @@
+import Prelude
+import UIKit
+
+private struct Margin {
+  static let leftRight: CGFloat = 8
+  static let topBottom: CGFloat = 15
+}
+
+class SettingsTextInputCell: UITableViewCell {
+  // MARK: - Accessors
+
+  private lazy var stackView: UIStackView = {
+    let layoutMargins = UIEdgeInsets.init(topBottom: Margin.topBottom, leftRight: Margin.leftRight)
```

Comment:
Compiler couldn't infer simple `.init` on my machine, so I had to use `UIEdgeInsets.init` explicitly